cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/RepositorySourceControlMetadataStore.java

```diff
@@ -58,14 +70,87 @@ private StructuredTable getRepositorySourceControlMetadataTable() {
+   * @throws IOException If an I/O error occurs while scanning the metadata records.
+   */
+  public int scan(ScanSourceControlMetadataRequest request,
+      String type, Consumer<SourceControlMetadataRecord> consumer) throws IOException {
+    Range.Bound startBound = Range.Bound.INCLUSIVE;
+    final Range.Bound endBound = Range.Bound.INCLUSIVE;
+    Collection<Field<?>> startFields = new ArrayList<>();
+
+    startFields.add(
+        Fields.stringField(StoreDefinition.RepositorySourceControlMetadataStore.NAMESPACE_FIELD,
+            request.getNamespace())
+    );
+    startFields.add(
+        Fields.stringField(StoreDefinition.RepositorySourceControlMetadataStore.TYPE_FIELD,
+            type)
+    );
+
+    if (request.getFilter().getSyncStatus() != null) {
+      startFields.add(
+          Fields.booleanField(StoreDefinition.RepositorySourceControlMetadataStore.IS_SYNCED_FIELD,
+              request.getFilter().getSyncStatus().toString().equals("SYNCED")));
+    }
+
+    Collection<Field<?>> endFields = startFields;
+
+    if (request.getScanAfter() != null) {
+      startBound = Range.Bound.EXCLUSIVE;
+      startFields =
+          request.getSortOn() == SortBy.LAST_SYNCED_DATE ? getLastModifiedRangeFields(request, type)
+              : getRangeFields(request, type);
+    }
+
+    Range range;
+    if (request.getSortOrder() == SortOrder.ASC) {
+      range = Range.create(startFields, startBound, endFields, endBound);
+    } else {
+      range = Range.create(endFields, endBound, startFields, startBound);
+    }
+
+    StructuredTable table = getRepositorySourceControlMetadataTable();
+    int limit = request.getLimit();
+    int count = 0;
+    try (CloseableIterator<StructuredRow> iterator = getScanIterator(table,
+        range,
+        request.getSortOrder(), request.getSortOn())
+    ) {
+      while (iterator.hasNext() && limit > 0) {
+        StructuredRow row = iterator.next();
+        SourceControlMetadataRecord record = decodeRow(row);
+        boolean nameContainsFilterMatch =
+            request.getFilter().getNameContains() == null || record.getName().toLowerCase()
+                .contains(request.getFilter().getNameContains().toLowerCase());
+
+        if (nameContainsFilterMatch) {
+          consumer.accept(record);
+          limit--;
+          count++;
+        }
+      }
+    }
+    return count;
+  }
+
   /**
    * Writes or updates the source control metadata for the specified repository metadata ID.
    *
-   * @param appRef The {@link ApplicationReference} for which source control metadata needs to be
-   *                       written or updated.
-   * @param isSynced       A boolean indicating whether the entity in repository is synced with the
-   *                       one in namespace
-   * @param lastSynced     The timestamp of the last git pull/push of the entity
+   * @param appRef     The {@link ApplicationReference} for which source control metadata needs to
```

Comment:
revert this change please

Reply:
Actually, this is a fix of checkstyle indentation warning